activation/activation.go

```diff
@@ -12,106 +12,109 @@ import (
 	"sync/atomic"
 )
 
+// AtxProtocol is the protocol id for broadcasting atxs over gossip
 const AtxProtocol = "AtxGossip"
 
 var activesetCache = NewActivesetCache(1000)
 
-type MeshProvider interface {
+// meshProviderMock
+type meshProvider interface {
 	GetOrphanBlocksBefore(l types.LayerID) ([]types.BlockID, error)
 	LatestLayer() types.LayerID
 }
 
-type EpochProvider interface {
-	Epoch(l types.LayerID) types.EpochId
-}
-
-type Broadcaster interface {
+type broadcaster interface {
 	Broadcast(channel string, data []byte) error
 }
 
-type PoETNumberOfTickProvider struct {
+type poETNumberOfTickProvider struct {
```

Comment:
this is very confusing... Let's consider poet a word and make the `ET` lowercase?